W/WIGXJPF/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WIGXJPF"
+version = v"1.13.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://fy.chalmers.se/subatom/wigxjpf/wigxjpf-latest.tar.gz", "90ab9bfd495978ad1fdcbb436e274d6f4586184ae290b99920e5c978d64b3e6a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/wigxjpf-1.13
+
+mkdir -p $WORKSPACE/destdir/lib
+mkdir -p $WORKSPACE/destdir/bin
+
+if [ "$(uname)" == "Darwin" ]; then
+    make lib/libwigxjpf_shared.dylib
+    cp lib/libwigxjpf_shared.dylib $WORKSPACE/destdir/lib/  
+else
+    make lib/libwigxjpf_shared.so
+    cp lib/libwigxjpf_shared.so $WORKSPACE/destdir/lib/
+    cp lib/libwigxjpf_shared.so $WORKSPACE/destdir/bin/libwigxjpf_shared.dll                    
+fi
```

Comment:
```suggestion

# Patch the CMake workflow to fix the install directories and build the libraries as shared libraries
echo -e "include(GNUInstallDirs)\n$(cat CMakeLists.txt)" > CMakeLists.txt
sed -i 's/STATIC//' CMakeLists.txt

mkdir -p build && cd build

cmake -B . -S .. \
    -DCMAKE_INSTALL_PREFIX=${prefix} \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
    -DCMAKE_BUILD_TYPE=Release \
    -DBUILD_SHARED_LIBS=ON

cmake --build . --parallel ${nproc}
cmake --install .
```

I think this should work to enable a CMake build of this without needing to actually have special patch files.